src/ucp/core/ucp_ep.inl

```diff
@@ -273,4 +273,12 @@ ucp_ep_config_connect_p2p(ucp_worker_h worker,
            !ucp_worker_is_tl_2iface(worker, rsc_index);
 }
 
+static UCS_F_ALWAYS_INLINE int ucp_ep_can_do_keepalive(ucp_ep_h ep)
```

Comment:
"can" and "supported & enabled" are diff things.. maybe rename smth like `ucp_ep_is_keepalive_enabled` ?

Reply:
ok, renamed